tests/test_util.py

```diff
@@ -34,6 +37,20 @@ def test_encode_uri():
     assert expected, encode_uri(uri)
 
 
+def test_ensuredir():
+    with tempfile.TemporaryDirectory() as tmp_path:
+        # Does not raise an exception.
```

Comment:
Could you add comment about purpose of the case like "does not raise an exception for existing directory" please?